app/view_models/sign_up_completions_show.rb

```diff
@@ -22,11 +22,7 @@ def initialize(loa3_requested:, decorated_session:, current_user:)
 
   # rubocop:disable Rails/OutputSafety
   def heading
-    safe_join([I18n.t(
-      'titles.sign_up.completion_html',
-      accent: content_tag(:strong, I18n.t("titles.sign_up.#{requested_loa}")),
-      app: APP_NAME
-    ).html_safe])
+    @current_user.decorate.identity_verified? ? verified_heading : loa_heading
```

Comment:
I've done a little digging, and I can't figure out how an LOA3 user reaches the sign up completions view without a verified identity. Can you help me understand why this conditional needs to check for a verified identity here?

Reply:
@jmhooper if the user goes directly to login.gov, and then verifies, by say entering the code from a letter. They don't have loa request == loa3' in their session. So they see the original message "you have created an account", when we want to show that they have just completed their verification. "You have verified your identity"

what im not sure about is if there is ever a situation where we want to show them the old loa3 message. So it may be worth taking that out